[Console] Deprecating Command getDefaultName and getDefaultDescription methods (#59565)
symfony/symfony: UPGRADE-7.3.md

   });
   ```

>  * Static methods `Command::getDefaultName()` and `Command::getDefaultDescription()` are deprecated.
>    Extract the command name and description through class reflection instead

Reviewer: ```suggestion
 * Deprecate methods `Command::getDefaultName()` and `Command::getDefaultDescription()` in favor of the `#[AsCommand]` attribute
```
